### ISSUE TITLE
manifest v3 compatibility

### ISSUE THREAD

**Issue by materemias** [joaomgcd/JoinChrome#128]
manifest v2 will soon be deprecated in chrome, I have not found any news related to Join's future in v3. Can you make the extension work on v3? what is the plan?

**Comment by joaomgcd** [comment]
Yes, I do want to make it v3 compatible soon. It's just a lot of work unfortunately...
